src/libos/src/fs/file.rs

```diff
@@ -95,6 +95,10 @@ pub trait File: Debug + Sync + Send + Any {
         return_op_unsupported_error!("fallocate")
     }
 
+    fn fs(&self) -> Result<Arc<dyn FileSystem>> {
```

Comment:
Is it possible to return `Result<&Arc<dyn FileSystem>>`? This way, the caller can decide whether a clone on `Arc` is needed, saving unnecessary clones.

Reply:
If return the reference, the inode_file struct have to store the Arc<dyn Filesystem> as a member. Do you think it makes sense?